.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+# Enable tokio's unstable feature set: https://docs.rs/tokio/latest/tokio/#unstable-features
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
What do we use this for?

Reply:
I thought we needed this to get the `JoinSet` task IDs, but I was wrong. Removing, thanks!